subsys/bluetooth/mesh/proxy_msg.c

```diff
@@ -0,0 +1,249 @@
+/*  Bluetooth Mesh */
```

Comment:
Would "gatt_msg.c" be more appropriate? I know this is just following the original suggestion though :)

Reply:
Should we follow the definition in spec?

Reply:
What does the definition in the specification mean to me is that the pb-gatt message borrows the definition of the proxy format? So this is my point of view, may I ask @jhedberg  point of view?